main.py

```diff
@@ -30,13 +31,14 @@ def main(is_simulation: bool, real_servo: bool) -> None:
     sim_time_start = time.time()
 
     if is_simulation:
-        imu = MockIMU(SIMULATION_LOG_PATH, real_time_simulation=True)
+        imu = MockIMU(SIMULATION_LOG_PATH, real_time_simulation=True, start_after_log_buffer=True)
         servo = Servo(SERVO_PIN) if real_servo else Servo(SERVO_PIN, pin_factory=MockFactory(pin_class=MockPWMPin))
+        logger = MockLogger(LOGS_PATH, delete_log_file=True)
```

Comment:
maybe make this a command line argument, that defaults to deleting the file. make it whatever you want -- e.g. k for keep file

Reply:
I think I'm going to use argparse now. I also want to add another argument on whether to run the mock sim in real time or not.